cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -417,8 +421,22 @@ public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanc
   @Override
   public Connection getQueryClient(Id.Namespace namespace) throws Exception {
     ConnectionConfig connConfig = clientConfig.getConnectionConfig();
-    String url = String.format("%s%s:%d?namespace=%s", Constants.Explore.Jdbc.URL_PREFIX, connConfig.getHostname(),
-                               connConfig.getPort(), namespace.getId());
+
+    Map<String, String> connParams = new HashMap<>();
+    connParams.put(ExploreConnectionParams.Info.NAMESPACE.getName(), namespace.getId());
+    AccessToken accessToken = clientConfig.getAccessToken();
+    if (accessToken != null) {
+      connParams.put(ExploreConnectionParams.Info.EXPLORE_AUTH_TOKEN.getName(), accessToken.getValue());
+    }
+    connParams.put(ExploreConnectionParams.Info.SSL_ENABLED.getName(),
+                   Boolean.toString(clientConfig.getConnectionConfig().isSSLEnabled()));
+    connParams.put(ExploreConnectionParams.Info.VERIFY_SSL_CERT.getName(),
+                   Boolean.toString(clientConfig.isVerifySSLCert()));
+
+    String connParamsString = Joiner.on("&").withKeyValueSeparator("=").join(connParams);
```

Comment:
nit: better to not include datatype in variable name. can just use the ``Joiner`` in the ``String.format`` below, right?